src/BoundaryValueDiffEq.jl

```diff
@@ -23,17 +23,9 @@ import SciMLBase: AbstractDiffEqInterpolation, StandardBVProblem, __solve, _unwr
 @reexport using ADTypes, DiffEqBase, NonlinearSolve, OrdinaryDiffEq, SparseDiffTools,
                 SciMLBase
 
-include("types.jl")
-include("utils.jl")
-include("algorithms.jl")
-include("alg_utils.jl")
-include("sparse_jacobians.jl")
-include("default_nlsolve.jl")
-
-function __solve(prob::BVProblem, alg::BoundaryValueDiffEqAlgorithm, args...; kwargs...)
-    cache = init(prob, alg, args...; kwargs...)
-    return solve!(cache)
-end
+include("../lib/BoundaryValueDiffEqCore/src/BoundaryValueDiffEqCore.jl")
+using ..BoundaryValueDiffEqCore
+include("../lib/BoundaryValueDiffEqCore/src/algorithms.jl")
```

Comment:
why is this needed?

Reply:
It's the wrapper algorithms in ODEInterfacExt require the basic `BoundaryVallueDiffAlgorithm` types to subtype wrapper algorithms

Reply:
but this should be removed by importing from core?

Reply:
yeah, it can be removed